src/main/java/com/github/pedbertor/gol/controller/ControllerBuilder.java

```diff
@@ -106,6 +107,17 @@ public ControllerBuilder setCellStateProcessor(CellStateProcessor cellStateProce
         return this;
     }
 
+    /**
+     * Sets console output state.
+     *
+     * @param isConsoleOutputEnabled the cell state processor
+     * @return a reference to this {@code ControllerBuilder} object
+     */
+    public ControllerBuilder setIsConsoleOutputEnabled(boolean isConsoleOutputEnabled) {
```

Comment:
@Sumlarys I believe we should follow the naming conventions for setters of boolean instance variables. For example, in this case I would use _setConsoleOutputEnabled(boolean consoleOutputEnabled)_ instead.